ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingConfig.java

```diff
@@ -201,6 +207,24 @@ public OptimizingConfig setBaseHashBucket(int baseHashBucket) {
     return this;
   }
 
+  public long getBaseMaxDelay() {
+    return baseMaxDelay;
+  }
+
+  public OptimizingConfig setBaseMaxDelay(long baseMaxDelay) {
+    this.baseMaxDelay = baseMaxDelay;
```

Comment:
could this be named as base.max.freshness?

Reply:
for the TableProperty, I will change it to `self-optimizing.trigger.base.freshness.max-seconds`.

Reply:
I changed it to `base.refresh-interval`